packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -195,6 +195,82 @@ enum AnimationBehavior {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=freeform}
```

Comment:
This code can probably be simplified by using stateful_widget_material_ticker as template?

Reply:
ok